assignments/callbacks.js

```diff
@@ -25,27 +25,63 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
+function gettingLength(arr, cb){
+  return cb(arr);
   // getLength passes the length of the array into the callback.
 }
 
+function itemsLength(items){
+  return items.length;
+}
+console.log(gettingLength(items, itemsLength));
+
 function last(arr, cb) {
+  return cb(arr);
   // last passes the last item of the array into the callback.
 }
 
+function lastArr(items){
+  return items.length - 1;
+}
+
+// console.log(last(items, lastArr));
+
+
 function sumNums(x, y, cb) {
+  return cb(x + y);
```

Comment:
this will need to be cb(x, y) if it will match the add function that you have below. Think of this line as an invocation of the function that you wrote below...number of parameters need to match and the order matters in some cases